tests/Parameters/PublishRecordingsParametersTest.php

```diff
@@ -29,19 +29,22 @@
  */
 class PublishRecordingsParametersTest extends TestCase
 {
-    public function testPublishRecordingsParameters()
+    public function testPublishRecordingsParameters(): void
     {
-        $recordingId      = $this->faker->uuid;
-        $publish          = $this->faker->boolean(50);
+        $recordingId    = $this->faker->uuid;
+        $publish        = $this->faker->boolean(50);
+        $newRecordingId = $this->faker->uuid;
```

Comment:
I thought that it would give enough meaning ;-) What do you have in mind?

I adapted to this:
```php
    public function testPublishRecordingsParameters()
    {
        $recordingId1 = $this->faker->uuid;
        $publish1     = $this->faker->boolean(50);
        $recordingId2 = $this->faker->uuid;
        $publish2     = !$publish1;

        $publishRecording = new PublishRecordingsParameters($recordingId1, $publish1);

        $this->assertEquals($recordingId1, $publishRecording->getRecordingId());
        $this->assertEquals($publish1, $publishRecording->isPublish());

        // Test setters that are ignored by the constructor
        $publishRecording->setRecordingId($recordingId2);
        $publishRecording->setPublish($publish2);
        $this->assertEquals($recordingId2, $publishRecording->getRecordingId());
        $this->assertEquals($publish2, $publishRecording->isPublish());
    }
```

Do you agree?

Reply:
This version implemented in new commit.